RecoJets/JetProducers/plugins/VirtualJetProducer.cc

```diff
@@ -228,8 +228,6 @@ VirtualJetProducer::VirtualJetProducer(const edm::ParameterSet& iConfig) {
 		fjRangeDef_ = RangeDefPtr( new fastjet::RangeDefinition(rhoEtaMax) );
 	} 
 
-	if( doFastJetNonUniform_ ) puCenters_ = iConfig.getParameter<vector<double> >("puCenters"); 	/// where is this used? 
```

Comment:
ok, done. Here is the runTheMatrix file: https://cernbox.cern.ch/index.php/s/AZxTXWLOsyxWtaO